webapp/src/components/App/Singlestudy/explore/Debug/Data/Text.tsx

```diff
@@ -111,24 +123,31 @@ function Text({ studyId, filePath, filename, canEdit }: DataCompProps) {
                 onUploadSuccessful={handleUploadSuccessful}
               />
             )}
-            <DownloadButton onClick={handleDownload} />
-          </Menubar>
-          <Box sx={{ overflow: "auto" }}>
-            <SyntaxHighlighter
-              style={atomOneDark}
-              lineNumberStyle={{
-                opacity: 0.5,
-                paddingRight: theme.spacing(3),
-              }}
-              customStyle={{
-                margin: 0,
-                padding: theme.spacing(2),
-                borderRadius: theme.shape.borderRadius,
-                fontSize: theme.typography.body2.fontSize,
-              }}
-              {...getSyntaxProps(text)}
+            <DownloadButton
+              onClick={handleDownload}
+              disabled={isEmptyContent(text)}
             />
-          </Box>
+          </Menubar>
+          {isEmptyContent(text) ? (
+            <EmptyView icon={GridOffIcon} title={t("study.results.noData")} />
```

Comment:
It can be confused to display a message for an empty file (no data can be undestand as there is not file at all), display nothing for me (or ask Alexander)

Reply:
before:
![image](https://github.com/user-attachments/assets/17683d4d-fa80-4381-8368-f491eb94cf18)



after the changes:
![image](https://github.com/user-attachments/assets/7d064a7d-e08c-4ba7-ae4b-718c726de26c)


it makes sense to handle this state, I think its fine 
you're right about the message that can be maybe more explicit like "empty file content" but does not make a huge difference in my opinion

Reply:
Better to ask Alexander to prevent a feedback

Reply:
**Update:** you are not wrong but the current debug view does not allow to edit text files, when it will be the case we should set the text editor to a fixed height and allow edition even if the file is empty. Currently it does not make sense to display an empty block